addons/web_editor/controllers/main.py

```diff
@@ -174,6 +175,10 @@ def _update_checklist_recursive (self, li, checked, children=False, ancestors=Fa
 
         return True
 
+    @http.route('/web_editor/video_url/data', type='json', auth='user', website=True)
+    def video_url_data(self, video_url, **kwargs):
+        return get_video_url_data(video_url, **kwargs)
```

Comment:
FWIW there is no validation anywhere here so any authenticated user (including portal) can trigger a fetch, and activity from our systems.

Reply:
Adding a check for internal user, you are right :+1: There is probably a way to be more precise than this but currently, with the new editor, all connected user (including portal ones) have access to the video upload UI (in project pad too for example) so kinda hard to do what is best is as this is being reviewed/fixed. Being internal is probably reasonable for this controller anyway.